tests/patterns/cp_exception.py

```diff
@@ -0,0 +1,5 @@
+def f():
```

Comment:
@amchiclet can you move this test file under tests/patterns/python/ so it does not clutter the tests/patterns/ directory?
(now actually OSS/tests/patterns/).